api/validator.js

```diff
@@ -39,6 +39,7 @@ const SCHEMA_JOB_PERMUTATION = Joi.object()
         secrets: Job.secrets,
         settings: Job.settings,
         sourcePaths: Job.sourcePaths,
+        stages: Base.stages,
```

Comment:
Job cannot be part of multiple stages.
This schema needs to be fixed so that a job points to a single stage.